### PR TITLE
fixes some stuff you forgot about in the new admin verb panel

### PR DESCRIPTION
**Pull request by scriptis**
## About The Pull Request

oversights from #72407

- re-adds the pod launcher button (you just kind of forgot about it)
- makes the admin verbs actually use the pointer cursor, you know, since they're buttons
- renames "query text" to "sqdl2 query"
## Why It's Good For The Game

90% of my ~~player-facing grief~~ admin events use drop pods
## Changelog
:cl:
fix: added missing pod launcher, spawn atom, fax panel, message pda, painting manager, trophy manager, shuttle manipulator, make pai, view all circuits verbs
fix: verb links now behave like links when you hover over them (ft. the oldest ie6 hack i know)
admin: "query text" is now "sqdl2 query"
/:cl:

### CODE REVIEW

**Comment by scriptis** [comment]
hold on you forgot about some other stuff like the fax panel let me get those back in

**Comment by scriptis** [comment]
ctrl+shift+f is your friend

this is now good to go. off to fix the other stuff that broke

**Comment by scriptis** [comment]
ugggggggh

**Comment by scriptis** [comment]
@ZephyrTFA if you can give this a quick OK this should be good to go

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm [hunk 1]
```diff
@@ -198,7 +198,7 @@
 		state = SDQL2_STATE_ERROR;\
 		CRASH("SDQL2 fatal error");};
 
-ADMIN_VERB(debug, query_text, "Query Text", "", R_DEBUG, query_text as message)
+ADMIN_VERB(debug, sdql2_query, "SDQL2 Query", "Run an SDQL2 query.", R_DEBUG, query_text as message)
```
Comment (ZephyrTFA) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm:201]:
```suggestion
ADMIN_VERB(debug, sdql2_query, "SDQL2 Query", "Run a SDQL2 Query", R_DEBUG, query_text as message)
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm (hunk 1 above)
Comment (scriptis) [reply code/modules/admin/verbs/SDQL2/SDQL_2.dm:201]: "a ess" versus "an ess"; the latter is correct

Comment (ZephyrTFA) [reply code/modules/admin/verbs/SDQL2/SDQL_2.dm:201]: I'm not really stickler for grammatical stuff, that was my only nitpick

**Review by ZephyrTFA — APPROVED** [review]

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm (hunk 1 above)
Comment (scriptis) [reply code/modules/admin/verbs/SDQL2/SDQL_2.dm:201]: sick, we're good to go then

**Comment by ZephyrTFA** [comment]
Reverting AVD into a project branch so I can finish it in its entirety (minor miscommunication happened)

**Comment by scriptis** [comment]
closing due to the above

**Pull request closed without merging**